editor/src/core/shared/template-path.ts

```diff
@@ -441,23 +441,32 @@ export function parentPath(path: InstancePath): TemplatePath
 export function parentPath(path: TemplatePath): TemplatePath | null
 export function parentPath(path: TemplatePath): TemplatePath | null {
   if (isScenePath(path)) {
-    return null
+    return null // FIXME This is wrong!
```

Comment:
But fine for now?

Reply:
Ah yes, the problem here is that since scene paths are also element paths, they can still have parents. I was considering handling this during this work but decided against since that would mean the PR would be addressing separate, unrelated issues, and especially since this work has absolutely highlighted that we should remove the concept of `ScenePath` and `InstancePath`